pkgs/by-name/sp/sportstracker/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  jdk21,
+  openjfx21,
+  maven,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  wrapGAppsHook3,
+  gtk3,
+}:
+
+let
+  jdkWithJFX = jdk21.override {
+    enableJavaFX = true;
+    openjfx21 = openjfx21.override { withWebKit = true; };
+  };
```

Comment:
On Darwin this should probably just be `jdk21.override { enableJavaFX = true; }` (as is it breaks nixpkgs-review, but not GHA Eval).
My quick fix was to just throw if it isn't a real openjdk: https://github.com/NixOS/nixpkgs/pull/366232/files